### PR TITLE
move pathplanning to litterate #74

### PR DESCRIPTION
**Pull request by JulienCalbert**
Move pathplanning to litterate #74 , also add Abstraction module into Dionysos module #67

### CODE REVIEW

File: docs/Project.toml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 CDDLib = "3391f64e-dcde-5f30-b752-e11513730f60"
 Cbc = "9961bab8-2fa3-5c5a-9d89-47fab24efd76"
 Colors = "5ae59095-9a9b-59fe-a467-6f913c188581"
+Dionysos = "d92c97cf-b87d-42c1-a9c0-25df00b4d958"
```
Comment (blegat) [inline docs/Project.toml:5]: Remove, it's ok if it's in the `Project.toml` of your local computer but don't push it.

**Comment by blegat** [comment]
Looks like installing PyPlot is failing: https://github.com/dionysos-dev/Dionysos.jl/pull/86/checks?check_run_id=1870352925#step:6:164
Moving to Plots should fix the issue although there might be a way to make PyPlot work on Github Actions

**Comment by egidioln** [comment]
> Looks like installing PyPlot is failing: https://github.com/dionysos-dev/Dionysos.jl/pull/86/checks?check_run_id=1870352925#step:6:164
> Moving to Plots should fix the issue although there might be a way to make PyPlot work on Github Actions


In the logs they suggest this. Maybe it's worth trying, no?
```
One alternative is to re-configure PyCall to use a different Python 
version on your system: set ENV["PYTHON"] to the path/name of the python
executable you want to use, run Pkg.build("PyCall"), and re-launch Julia.

Another alternative is to configure PyCall to use a Julia-specific Python
distribution via the Conda.jl package (which installs a private Anaconda
Python distribution), which has the advantage that packages can be installed
and kept up-to-date via Julia.  As explained in the PyCall documentation,
set ENV["PYTHON"]="", run Pkg.build("PyCall"), and re-launch Julia. Then,
To install the matplotlib module, you can use `pyimport_conda("matplotlib", PKG)`,
where PKG is the Anaconda package that contains the module matplotlib,
or alternatively you can use the Conda package directly (via
`using Conda` followed by `Conda.add` etcetera).
```

**Comment by blegat** [comment]
Yes, it might be possible by mimicking what PyPlot.jl does:
https://github.com/JuliaPy/PyPlot.jl/blob/master/.travis.yml#L8-L16
However, since we're going to migrate the PyPlot code to Plots and then to RecipesBase in the future, we might just go with the long-term solution. I guess when we use Plots, it used GR by default which works out of the box on CI.

**Comment by egidioln** [comment]
> Yes, it might be possible by mimicking what PyPlot.jl does:
> https://github.com/JuliaPy/PyPlot.jl/blob/master/.travis.yml#L8-L16
> However, since we're going to migrate the PyPlot code to Plots and then to RecipesBase in the future, we might just go with the long-term solution. I guess when we use Plots, it used GR by default which works out of the box on CI.

Indeed! That's a better solution =)

**Comment by JulienCalbert** [comment]
so for the moment we can suppress code calls in the path planning display section ?

**Comment by blegat** [comment]
Yes, let's start with removing the plots. Also remove `PyPlot` from the `Project.toml`

**Review by blegat — APPROVED** [review]

**Pull request merged**